addons/kestrel4500/functions/fnc_createKestrelDialog.sqf

```diff
@@ -20,7 +20,7 @@ if (underwater ACE_player) exitWith { false };
 if (!("ACE_Kestrel4500" in (uniformItems ACE_player)) && !("ACE_Kestrel4500" in (vestItems ACE_player))) exitWith { false };
 
 GVAR(Overlay) = false;
-3 cutText ["", "PLAIN"];
+(QGVAR(Layer) call BIS_fnc_rscLayer) cutText ["", "PLAIN"];
```

Comment:
```suggestion
QGVAR(Layer) cutText ["", "PLAIN"];
```

See: https://community.bistudio.com/wiki/cutRsc#Alternative_Syntax_2

etc.

Reply:
Copy, was thinking just using the QGVAR(), but had a look at how it was done in MicroDAGR:

https://github.com/acemod/ACE3/blob/master/addons/microdagr/functions/fnc_openDisplay.sqf#L46

Can strip the `BIS_fnc_rscLayer` call if preferred.

Reply:
`BIS_fnc_rscLayer` is obsolete. It is now replaced with the engine string ids. The function only exists for bwc and all it does now is call a cutX command with the layer:
<details><summary>BIS_fnc_rscLayer</summary>

```sqf
scopeName "function";

params [["_layer", ""]];



if !(_layer isEqualType "") exitWith {[_layer, "isEqualType", ""] call (missionNamespace getVariable "BIS_fnc_errorParamsType")};

if (_layer isEqualTo "") then
{
    if (_fnc_scriptNameParent isEqualTo _fnc_scriptName) then
    {
        "RSC Layer name cannot be empty string" call BIS_fnc_error;
        -1 breakOut "function";
    };

    _layer = _fnc_scriptNameParent;
};

private _id = allCutLayers find _layer;
if (_id < 0) exitWith {_layer cutFadeOut 0};

_id
```

</details>

MicroDAGR just uses the outdated stuff.